Chummer/Program.cs

```diff
@@ -150,10 +150,14 @@ static void Main()
                 try
                 {
                     LogManager.ThrowExceptions = true;
-                    ConfigurationItemFactory.Default.Targets.RegisterDefinition(
-                        "ApplicationInsightsTarget",
-                        typeof(Microsoft.ApplicationInsights.NLogTarget.ApplicationInsightsTarget)
-                    );
+                    if (GlobalOptions.UseLoggingApplicationInsights != EnumUseLoggingApplicationInsights.onlylocal)
```

Comment:
I don't think this is the intended operator, did you intend `>` (`>=EnumUseLoggingApplicationInsights.onlymetric`)?